maps/away/ships/dominia/dominian_science_vessel/dominian_science_vessel_ghostroles.dm

```diff
@@ -0,0 +1,133 @@
+	uniform = /obj/item/clothing/under/dominia/imperial_suit/volvalaad
+	shoes = /obj/item/clothing/shoes/laceup
+	back = /obj/item/storage/backpack/satchel
+	id = /obj/item/card/id/house_volvalaad
+
+	l_ear = /obj/item/device/radio/headset/ship
+
+	backpack_contents = list(/obj/item/storage/box/survival = 1)
+
+/datum/outfit/admin/house_volvalaad_officer/get_id_access()
+	return list(ACCESS_HOUSE_VOLVALAAD_SHIP, ACCESS_EXTERNAL_AIRLOCKS)
+
+
+/datum/ghostspawner/human/house_volvalaad_armsman
+	short_name = "house_volvalaad_armsman"
+	name = "House Volvalaad Armsman"
+	desc = "You are a Ma’zal who serves as an Imperial Fleet armsman aboard a House Volvalaad science vessel, and are expected to defend the ship from boarders. Armsmen are experienced enlisted personnel of the Imperial Fleet who are often recruited from the Imperial Army. While not one of the elite Marines you are well-trained and well-equipped to defend their vessel. Armsmen are often older and more experienced than voidsmen and many have continued to serve in order to pay off the Mo’ri’zal (“Blood Debt,” the Empire’s form of taxation) of their relatives. You have been trained to obey your officer, and most Primaries, without question or hesitation. (OOC Note: Players should be familiar with Dominian lore and play a character with a background appropriate to a Dominian Ma'zal.)"
+	mob_name_prefix = "ARMSN. " //Armsman
+	culture_restriction = list(/singleton/origin_item/culture/dominia)
+
+	spawnpoints = list("house_volvalaad_armsman")
+	max_count = 2
+
+	outfit = /datum/outfit/admin/house_volvalaad_armsman
+
+	assigned_role = "House Volvalaad Armsman"
+	special_role = "House Volvalaad Armsman"
+
+/datum/outfit/admin/house_volvalaad_armsman
+	name = "House Volvalaad Armsman"
+	head = /obj/item/clothing/head/dominia/fleet/armsman
+	uniform = /obj/item/clothing/under/dominia/fleet/armsman
+	accessory = /obj/item/clothing/accessory/poncho/dominia_cape/mantle
+	shoes = /obj/item/clothing/shoes/jackboots
+	back = /obj/item/storage/backpack/satchel
+	id = /obj/item/card/id/house_volvalaad
+
+	l_ear = /obj/item/device/radio/headset/ship
+
+	backpack_contents = list(/obj/item/storage/box/survival = 1)
+
+/datum/outfit/admin/house_volvalaad_armsman/get_id_access()
+	return list(ACCESS_HOUSE_VOLVALAAD_SHIP, ACCESS_EXTERNAL_AIRLOCKS)
+
+/datum/ghostspawner/human/house_volvalaad_scientist
+	short_name = "house_volvalaad_scientist"
+	name = "House Volvalaad Scientist"
+	desc = "You are a Secondary or Ma'zal affiliated with House Volvalaad who serves as a scientist aboard a House Volvalaad science vessel. Visit planetary bodies, make new discoveries, and bring honor to the Goddess and House. (OOC Note: Players should be familiar with Dominian lore and play a character with a background appropriate to a Dominian Ma'zal or Secondary.)"
+	mob_name_prefix = null
+	culture_restriction = list(/singleton/origin_item/culture/dominia)
+
+	spawnpoints = list("house_volvalaad_scientist")
+	max_count = 2
```

Comment:
This is a science ship, should it not have more science crew, and less military crew? It has 2 scientists and 3-4 military crew, voidsmen/armsmen and...